geomesa-hbase/geomesa-hbase-datastore/src/test/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreTest.scala

```diff
@@ -151,6 +152,7 @@ class HBaseDataStoreTest extends Specification with LazyLogging {
         feature.getAttribute(attribute) mustEqual results.find(_.getID == feature.getID).get.getAttribute(attribute)
       }
     }
+    ds.getFeatureSource(typeName).getFeatures(query).size() mustEqual results.length
```

Comment:
may also want to compare against the original expected value incase results is returning the wrong answer though I assume our other unit tests for hbase query would catch that :+1: